vmc/resource_vmc_sddc.go

```diff
@@ -177,6 +167,10 @@ func resourceSddc() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"cluster_info": {
```

Comment:
is the SDDC documentation updated with cluster_info?

Reply:
Done